go.mod

```diff
@@ -34,3 +32,83 @@ require (
 	sigs.k8s.io/controller-tools v0.6.3-0.20210916130746-94401651a6c3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
```

Comment:
Is this a new thing for go modules in 1.17?

Reply:
seems like it, i ran `make vendor` and it produces this